projects/view/client/rtProject/rtProjectInfoSection.html

```diff
@@ -0,0 +1,45 @@
+<template name="rtProjectInfoSection">
```

Comment:
Finally, the two spaces standard is about compactness, while still making code easy to visually scan. Code that uses tab indentation, particularly in HTML templates with multiple nested elements, quickly flows out of the page (by convention, a 'page width' is around 80 characters).